packages/react/src/experimental/Skeleton/SkeletonBox.tsx

```diff
@@ -2,13 +2,16 @@ import type React from 'react'
 import styled, {keyframes} from 'styled-components'
 import sx, {type SxProp} from '../../sx'
 import {get} from '../../constants'
+import type {StyledBoxProps} from '../../Box/Box'
+import {background, border, color, flexbox, grid, layout, position, shadow, space, typography} from 'styled-system'
 
 type SkeletonBoxProps = {
   /** Height of the skeleton "box". Accepts any valid CSS `height` value. */
   height?: React.CSSProperties['height']
   /** Width of the skeleton "box". Accepts any valid CSS `width` value. */
   width?: React.CSSProperties['width']
-} & SxProp
+} & StyledBoxProps &
```

Comment:
Looks like a system prop addition, we don't prefer system props in the API anymore. We recommend using sx or css modules